src/js/components/AppPageComponent.jsx

```diff
@@ -439,12 +440,14 @@ var AppPageComponent = React.createClass({
       content = this.getTaskDetailComponent();
     }
 
+    var group = ViewHelper.getGroupFromAppId(state.appId);
+
     return (
       <div>
-        <AppBreadcrumbsComponent
-          activeTaskId={state.activeTaskId}
-          activeViewIndex={state.activeViewIndex}
-          appId={state.appId} />
+        <BreadcrumbComponent
+          group={group}
+          app={state.appId}
```

Comment:
I'd kinda prefer if we named the `app` parameter `appId` – I was confused thinking the BreadcrumbComponent would receive the whole app object... Maybe same goes for `taskId`. What do you think, @philipnrmn ?